src/main/java/com/magnatic/DemoApplication.java

```diff
@@ -0,0 +1,12 @@
+package com.magnatic;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class DemoApplication {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.10.48.216:8090/sonar/coding_rules#rule_key=squid%3AS1118)